conslogging/conslogging.go

```diff
@@ -107,21 +113,20 @@ func (cl ConsoleLogger) WithFailed(isFailed bool) ConsoleLogger {
 // PrintSuccess prints the success message.
 func (cl ConsoleLogger) PrintSuccess() {
 	cl.mu.Lock()
-	defer cl.mu.Unlock()
 	cl.color(successColor).Fprintf(cl.w, "=========================== SUCCESS ===========================\n")
+	cl.mu.Unlock()
 }
 
 // PrintFailure prints the failure message.
 func (cl ConsoleLogger) PrintFailure() {
 	cl.mu.Lock()
-	defer cl.mu.Unlock()
 	cl.color(warnColor).Fprintf(cl.w, "=========================== FAILURE ===========================\n")
+	cl.mu.Unlock()
```

Comment:
What's the reason for this change?

Reply:
This was me reverting from the attempt at using the global `printingMutex`. I'll revert these back to the defer calls. Sorry about this!